containenv/commands/containenv.py

```diff
@@ -219,3 +234,41 @@ def write_runcontainenv(self):
     def publish_image(self):
         '''if there's a shared registry publish to it'''
         pass
+
+
+class Activate(object):
+    '''Usage: containenv activate [help]
```

Comment:
Does this have weird indentation when you run `containenv activate -h`? I wouldn't expect `dedent` to do anything here.